### DISCUSSION TITLE
Supabase Integration - redirect_uri not allowed

### DISCUSSION THREAD

**Discussion by nate-kelley-buster** [supabase/supabase#21273] [Questions]
Hello! I am attempting to following [the instruction here](https://supabase.com/docs/guides/platform/oauth-apps/build-a-supabase-integration) on creating a supabase integration. Specifically, I am following the example code https://github.com/supabase/supabase/blob/master/examples/edge-functions/supabase/functions/connect-supabase/index.ts.

However, I keep running into issue when adjusting the `redirectUri` (line 11) from localhost to point to my actual uri. The error I am getting is: `{"message":"redirect_uri not allowed"}`.

Is anyone else getting this same issue?

**Comment by tgirotto** [comment]
@nate-kelley-buster I had the same problem, and fixed by going to Supabase dashboard -> account -> oauth apps and adding the uri to the list of allowed uris.
